processor/lsmintervalprocessor/internal/merger/value.go

```diff
@@ -110,6 +110,17 @@ func NewValue(resLimit, scopeLimit, metricLimit, datapointLimit config.LimitConf
 	}
 }
 
+// DatapointsCount returns the count of datapoint in the value representation.
```

Comment:
```suggestion
// DatapointsCount returns the count of datapoints in the value representation.
```

Reply:
I have a PR for adding parallel benchmarks: https://github.com/elastic/opentelemetry-collector-components/pull/350 - will add the results once that PR is merged